bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_buttons.html.erb

```diff
@@ -1,23 +1,31 @@
 <%
 stimulus_controller = 'fields--button-toggle'
 form ||= current_fields_form
+
+# Initialize options.
 html_options ||= {}
-html_options[:id] ||= form.field_id(method)
-multiple ||= false
+options ||= {}
 other_options ||= {}
-options ||= options_for(form, method)
+button_field_options ||= options_for(form, method)
+
+# For backwards compatibility.
+options.merge!(html_options)if options.is_a?(Hash)
+
+options[:id] ||= form.field_id(method)
+
+multiple ||= false
 %>
 
-<% content = render 'shared/fields/field', form: form, method: method, options: html_options, other_options: other_options do %>
+<% content = render 'shared/fields/field', form: form, method: method, options: options, other_options: other_options do %>
   <% content_for :field do %>
     <div>
-      <% options.each do |value, label| %>
+      <% button_field_options.each do |value, label| %>
         <% checked = form.object.send(method).is_a?(Array) ? form.object.send(method).map(&:to_s).include?(value.to_s) : form.object.send(method).to_s == value.to_s  %>
         <label class="btn-toggle" data-controller="<%= stimulus_controller %>">
           <% if multiple %>
-            <%= form.check_box method, {multiple: multiple, checked: checked, data: { "#{stimulus_controller}-target": 'shadowField' }}, value, "" %>
+            <%= form.check_box method, {multiple: multiple, checked: checked, data: { "#{stimulus_controller}-target": 'shadowField' }, **options.except(:id)}, value, "" %>
```

Comment:
I excluded `id` here and on line 28 because `id` is already automatically being populated.

When using multiple buttons, the ids look like this (considering the partial test from the starter repository):
`partial_test_single_button_test_one`
`partial_test_single_button_test_two`
`partial_test_single_button_test_three`

The tests were failing because `options[:id]` looks like this, and it was overriding the automatically created ids with three of the same id:
`partial_test_single_button_test`